client/state/selectors/get-store-referrers-by-date.js

```diff
@@ -14,6 +14,31 @@ import { sortBySales } from 'woocommerce/app/store-stats/referrers/helpers';
 
 export default function( state, { siteId, statType, query, endSelectedDate, limit, paginate } ) {
 	const rawData = getSiteStatsNormalizedData( state, siteId, statType, query );
-	const selectedData = find( rawData, d => d.date === endSelectedDate ) || { data: [] };
+	const allRaw = rawData.map( row => {
```

Comment:
This is a pretty substantial change to this selector logic, should we add some test cases?